flang/test/Lower/program-units-fir-mangling.f90

```diff
@@ -92,33 +92,6 @@ module function erase()
   end interface
 end module color_points
 
-submodule (color_points) color_points_a
```

Comment:
nit: Should these be commented out instead of deleted? It would be nice to bring them back once work on SUBMODULE support begins.

Reply:
Good idea.  Will do.